ui/build.js

```diff
@@ -489,6 +489,11 @@ function addClassToCSS(inputPath, outputPath,parentClass ) {
           if(node.children.first.type === 'ClassSelector' || node.children.first.name.startsWith('*')){
             classData.name+=' '
           }
+          // Classes starting with pf are already Perfetto specific classes
+          // Not adding this causes Perfetto popups which exist outside the normal Perfetto dom to not be styled
```

Comment:
nit: This second sentence confused me, but I think I finally got it. Can you please reword `Not adding this` to `Not skipping them here`